lib/igniter/project/application.ex

```diff
@@ -90,14 +90,13 @@ defmodule Igniter.Project.Application do
          true <- Igniter.Code.List.list?(zipper),
          {:ok, zipper} <- Igniter.Code.Keyword.get_key(zipper, :mod) do
       case Igniter.Code.Common.expand_literal(zipper) do
```

Comment:
This function can return `{:ok, app_module}` or `{:ok, {app_module, opts}}`, so we need to handle that.